wee_slack.py

```diff
@@ -3791,6 +3791,8 @@ def unwrap_attachments(message_json, text_before):
             # $author: (if rest of line is non-empty) $title ($title_link) OR $from_url
             # $author: (if no $author on previous line) $text
             # $fields
+            if 'original_url' in attachment and not config.link_previews:
+               return '' 
```

Comment:
We should do `continue` instead of `return`, in case there are more attachments.